### PR TITLE
Book page header

### PR DESCRIPTION
**Pull request by davidpmccormick**
Adding jauntily-boxed image to the book `PageHeader`

image

### CODE REVIEW

File: common/model/books.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// @flow
```
Comment (jamesgorrie) [inline common/model/books.ts:1]: No need?

File: common/model/books.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  subtitle: string | null,
+  orderLink: string | null,
+  price: string | null,
+  format: string | null,
+  extent: string | null,
+  isbn: string | null,
+  reviews: (Review[]) | null,
+  datePublished: Date | null,
+  authorName: string | null, // This is structuredText in Prismic >.<
+  authorImage: string | null, // This is a link in Prismic >.<
```
Comment (jamesgorrie) [inline common/model/books.ts:22]: Why do we have it as a string?

File: common/model/image.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import type { Tasl } from './tasl';
```
Comment (jamesgorrie) [inline common/model/image.ts:1]: Not sure we need to use import type in `TS`, it's just import.

File: common/model/books.ts (hunk 2 above)
Comment (davidpmccormick) [reply common/model/books.ts:22]: I'm guessing because `// This is a link in Prismic >.<`

File: common/model/image.ts (hunk 3 above)
Comment (davidpmccormick) [reply common/model/image.ts:1]: ah ok – good to know. So you can't have a component and type named the same exported from the same file?

**Review by jamesgorrie — APPROVED** [review]
Typescript!

**Pull request merged**